src/neoTrackerPanel.ts

```diff
@@ -165,9 +167,33 @@ export class NeoTrackerPanel implements INeoSubscription, INeoStatusReceiver {
                     ((this.viewState.currentBlock !== undefined) ? ActivePage.BlockDetail : ActivePage.Blocks );
             } else if (message.e === trackerEvents.Copy) {
                 await vscode.env.clipboard.writeText(message.c);
+            } else if (message.e === trackerEvents.Search) {
+                const asNumber = parseInt(message.c);
+                const block = await this.rpcConnection.getBlock((asNumber + '') === message.c ? asNumber : message.c, this);
```

Comment:
Add a !isNaN check so we skip the RPC call for values that are clearly not block indexes